internal/app/machined/internal/phase/upgrade/leave_etcd.go

```diff
@@ -7,17 +7,44 @@ package upgrade
 import (
 	"context"
 	"log"
+	"os"
 	"time"
 
 	"github.com/pkg/errors"
+	"github.com/talos-systems/talos/internal/app/machined/internal/phase"
+	"github.com/talos-systems/talos/internal/app/machined/internal/platform"
+	"github.com/talos-systems/talos/internal/app/machined/internal/runtime"
 	"github.com/talos-systems/talos/pkg/constants"
+	"github.com/talos-systems/talos/pkg/userdata"
 
 	"go.etcd.io/etcd/clientv3"
 	"go.etcd.io/etcd/pkg/transport"
 )
 
-// LeaveEtcd removes a member of etcd.
-func LeaveEtcd(hostname string) (err error) {
+// LeaveEtcd represents the task for stop all containerd tasks in the
```

Comment:
looks like comment doesn't relate to the task?

Reply:
Whoops. Thanks.